src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -0,0 +1,89 @@
+<Project>
+
+  <PropertyGroup>
+    <OutputInRepoRoot>true</OutputInRepoRoot>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="..\Microsoft.AspNetCore.App.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion)</TargetFramework>
+    <IsShippingPackage>true</IsShippingPackage>
+
+    <PackageDescription>This package provides reference assemblies for compiling an ASP.NET Core application. This package is an internal implementation of the .NET Core SDK and is not meant to be used as a normal PackageReference.</PackageDescription>
+    <PackageTags>aspnetcore;targeting-pack</PackageTags>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+
+    <!-- Subject to change: see https://github.com/dotnet/designs/pull/50 -->
+    <PackageType>TargetingPack</PackageType>
+    <RefAssemblyPackagePath>ref/$(TargetFramework)/</RefAssemblyPackagePath>
```

Comment:
@nguerrera what was the result of your battle with NuGet on this one? I had to step out early from that meeting on Monday.

Reply:
I don't really care either way, just wanted to know which to use to make this actually work with the SDK. Is the SDK being updated soon to look for `data/`? Last I checked, the FrameworkReference resolution still looks for `ref/<TFM>/`

Reply:
Anyway, I think we can follow up with a PR to these two lines when we have the decision.

Reply:
Ok, let me and @pakrym know when a decision is reached. I'll leave as `ref/<tfm>` for now cuz I want to test this with the SDK to find the holes in this implementation.